src/gui/mod.rs

```diff
@@ -142,6 +142,11 @@ impl QuickTagApp {
 
         quicktag_core::classes::load_schemafile();
 
+        // Check cache status and show warning if needed
+        if let Some(warning_msg) = check_cache_wordlist_status() {
+            TOASTS.lock().warning(warning_msg);
+        }
```

Comment:
Toasts are a bad way to display important messages like this, as they only show for a couple seconds, making it easy for the user to miss them